build/yaml/templates/npm-package-steps.yml

```diff
@@ -0,0 +1,29 @@
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
```

Comment:
I'm assuming this version is consistent with our other publishing pipelines for other JS software we own as a team?

Reply:
They use 10.x, 12.x, 14.x in parallel. Tracking at #565 and will follow up in another PR.